docs/index.adoc

```diff
@@ -424,10 +486,15 @@ creation of a new active wallet regardless of age.
 |depositTxMaxFee
 |The max amount of satoshis per deposit that the wallet is allowed to pay to miners.
 |Yes
-|`10000 sats`
+|`10000 satoshis`
 
 4+s|Redeeming
 
+|redemptionDustThreshold
+|The minimum amount of satoshis in a valid redemption.
+|Yes
+|`1000000 satoshis`
+
 |redemptionTreasuryFeeDivisor
 |The redemption fee divisor of one BTC to take as a treasury fee.
```

Comment:
I find the parameter description from `Bridge.sol` less confusing:
```suggestion
|A divisor used to compute the treasury fee taken from each redemption request and transferred to the treasury upon successful request finalization.
```
We could also consider adding info about how it's computed: `treasuryFee = requestedAmount / redemptionTreasuryFeeDivisor`.

Similar comment applies to `depositTreasuryFeeDivisor` description.

Reply:
[54b7a3a](https://github.com/keep-network/tbtc-v2/pull/346/commits/54b7a3aa103d9dfe4e3cce4542eb7b703710ba88)